### DISCUSSION TITLE
LM-Infinite

### DISCUSSION THREAD

**Discussion by bitnom** [vllm-project/vllm#932] [Ideas]
This looks fun: [https://www.youtube.com/watch?v=5XaJQKgL9Hs](https://www.youtube.com/watch?v=5XaJQKgL9Hs)
[![Meta AI LM-Infinite - Massive LLM improvement!](https://img.youtube.com/vi/5XaJQKgL9Hs/0.jpg)](https://www.youtube.com/watch?v=5XaJQKgL9Hs)
paper: https://huggingface.co/papers/2308.16137

**Comment by kyegomez** [comment]
Hey, I have a very basic implementation here
https://github.com/kyegomez/LM-Infinite
